network-api/networkapi/management/commands/flush_models.py

```diff
@@ -49,9 +48,6 @@ def handle(self, *args, **options):
         self.stdout.write('Dropping all Pages')
         WagtailPage.objects.exclude(id=1).delete()
 
-        self.stdout.write('Dropping all Products...')
```

Comment:
new buyersguide products are wagtail pages, and so already get removed by the line above